solc/CommandLineInterface.cpp

```diff
@@ -625,17 +625,39 @@ bool CommandLineInterface::parseArguments(int _argc, char const* const* _argv)
 	return true;
 }
 
+void CommandLineInterface::printLogo()
+{
+	if (!m_tty)
+		return;
+
+	// Simply print out the logo (with white background) onto stdout.
+	// The format is Sixel, which is part of the VT340 standard, is using DCS (device control string)
+	// to submit the data through stdout.
+	// Non-supporting terminals will simply ignore the DCS sequence, resulting into a no-op.
+	//
+	// The logo was converted from the docs/logo.svg using:
+	//     inkscape -w 128 -h 128 docs/logo.svg -o solc/logo.png
+	//     img2sixel solc/logo.png solc/logo.sixel
+	// Since the DCS payload consists exclusively of ASCII printable characters, I decided to embedd it
```

Comment:
Spellcheck complains:
```suggestion
	// Since the DCS payload consists exclusively of ASCII printable characters, I decided to embed it
```